src/Ormolu/Printer/Meat/Declaration/Class.hs

```diff
@@ -103,8 +103,9 @@ p_funDep (before, after) = do
 defltEqnToInstDecl :: TyFamDefltEqn GhcPs -> TyFamInstDecl GhcPs
 defltEqnToInstDecl FamEqn {..} = TyFamInstDecl {..}
   where
-    eqn = FamEqn {feqn_pats = tyVarsToTypes feqn_pats, ..}
+    eqn = FamEqn {feqn_pats = map HsValArg (tyVarsToTypes feqn_pats), ..}
```

Comment:
This is questionable. I don't have enough experience with GHC internals to say whether it's the right thing to do.